cmd/chain-maind/app/app.go

```diff
@@ -137,7 +137,24 @@ func initRootCmd(rootCmd *cobra.Command, encodingConfig params.EncodingConfig) {
 		if err != nil {
 			return err
 		}
+
+		tmpFilePath := cleanedPath + ".tmp"
+		tmpFile, err := os.Create(tmpFilePath)
+		if err != nil {
+			return err
+		}
+
 		defer func() {
```

Comment:
some comment `defer` , this line is run afterwards?

Reply:
Yes.